### ISSUE TITLE
Support for resuming from existing search record

### ISSUE THREAD

**Issue by gl387** [cornell-zhang/datuner#20]
DATuner should be able to read in a database of previously explored design points, and resume the search process that is aware of the history information.

**Comment by ChangXuSunny** [comment]
Yeah DATuner should read in the database to check whether searching is finished. But I am afraid the search process can't be aware of all the history information, since different search processes write different dbs. And the duplicated measurement still exists.
